src/resampler.rs

```diff
@@ -17,43 +18,16 @@ impl fmt::Display for Error {
             Error::InvalidArg => "Invalid argument.",
             Error::PtrOverlap => "Input and output buffers overlap.",
             Error::Overflow => "Muldiv overflow.",
+            Error::UnknownError => "Unknown error.",
```

Comment:
This seems unrelated.

Reply:
I added another error because I didn't want to keep the unreachable function.